posthog/models/raw_sessions/sessions_on_events_overrides.py

```diff
@@ -0,0 +1,754 @@
+    max_inserted_at SimpleAggregateFunction(max, DateTime64(6, 'UTC')),
+
+    -- urls
+    has_pageview_or_screen SimpleAggregateFunction(max, Boolean), -- this makes it not a subset of the sessions table, but is needed to handle url logic
+    entry_url AggregateFunction(argMin, Nullable(String), DateTime64(6, 'UTC')),
+    end_url AggregateFunction(argMax, Nullable(String), DateTime64(6, 'UTC')),
+    last_external_click_url AggregateFunction(argMax, Nullable(String), DateTime64(6, 'UTC')),
+
+    -- attribution
+    entry_referring_domain AggregateFunction(argMin, Nullable(String), DateTime64(6, 'UTC')),
+    entry_utm_source AggregateFunction(argMin, Nullable(String), DateTime64(6, 'UTC')),
+    entry_utm_campaign AggregateFunction(argMin, Nullable(String), DateTime64(6, 'UTC')),
+    entry_utm_medium AggregateFunction(argMin, Nullable(String), DateTime64(6, 'UTC')),
+    entry_utm_term AggregateFunction(argMin, Nullable(String), DateTime64(6, 'UTC')),
+    entry_utm_content AggregateFunction(argMin, Nullable(String), DateTime64(6, 'UTC')),
+    entry_gclid AggregateFunction(argMin, Nullable(String), DateTime64(6, 'UTC')),
+    entry_gad_source AggregateFunction(argMin, Nullable(String), DateTime64(6, 'UTC')),
+    entry_fbclid AggregateFunction(argMin, Nullable(String), DateTime64(6, 'UTC')),
+
+    -- for channel type calculation, it's enough to know if these were present
+    entry_has_gclid AggregateFunction(argMin, Boolean, DateTime64(6, 'UTC')),
+    entry_has_fbclid AggregateFunction(argMin, Boolean, DateTime64(6, 'UTC')),
+
+    -- for lower-tier ad ids, just put them in a map, and set of the ones present
+    entry_ad_ids_map AggregateFunction(argMin, Map(String, String), DateTime64(6, 'UTC')),
+    entry_ad_ids_set AggregateFunction(argMin, Array(String), DateTime64(6, 'UTC')),
+
+    -- bounce rate
+    page_screen_autocapture_uniq_up_to AggregateFunction(groupUniqArray(2), Nullable(UUID)),
+
+) ENGINE = {engine}
+"""
+
+new_line = "\n"
+
+# See https://kb.altinity.com/altinity-kb-queries-and-syntax/jsonextract-to-parse-many-attributes-at-a-time/
+# Or https://posthog.slack.com/archives/C02JQ320FV3/p1721406540313379?thread_ts=1721334861.073739&cid=C02JQ320FV3
+# if updating these, you'll want to make a migration to update the MV. Also update the sessions-on-events + overrides
+PROPERTIES = f"""
+        JSONExtract(properties, 'Tuple(
+            `$current_url` Nullable(String),
+            `$external_click_url` Nullable(String),
+            `$referring_domain` Nullable(String),
+            `utm_source` Nullable(String),
+            `utm_campaign` Nullable(String),
+            `utm_medium` Nullable(String),
+            `utm_term` Nullable(String),
+            `utm_content` Nullable(String),
+            `gclid` Nullable(String),
+            `gad_source` Nullable(String),
+            `fbclid` Nullable(String),
+{f',{new_line}'.join([f'            `{ad_id}` Nullable(String)' for ad_id in SESSION_V3_LOWER_TIER_AD_IDS])}
```

Comment:
If we need to update the values in SESSION_V3_LOWER_TIER_AD_IDS, do we need to do a migration?